components/MapView.js

```diff
@@ -502,9 +498,6 @@ class MapView extends React.Component {
         onMapReady: this._onMapReady,
         onLayout: this._onLayout,
       };
-      if (Platform.OS === 'ios' && ANDROID_ONLY_MAP_TYPES.includes(props.mapType)) {
-        props.mapType = MAP_TYPES.STANDARD;
-      }
```

Comment:
@gilbox good point! I can add this check back and base it off the platform and provider